src/panels/lovelace/editor/config-elements/hui-button-card-editor.ts

```diff
@@ -109,6 +95,31 @@ export class HuiButtonCardEditor extends LitElement
 
     const dir = computeRTLDirection(this.hass!);
 
+    if (this._editActionConfig) {
+      return html`
+        <hui-detail-editor-base
```

Comment:
Why is the rendering of the action editor not handled by `HuiActionBaseCardEditor`?

Reply:
Yeah, good idea